docker-entrypoint-setup-envars.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+set -eo pipefail
+
+# Find and replace environment variable placeholder names with their values at run time.
+echo "Replacing environment variables"
+replace() {
+  find $DEPLOYMENT_PATH/ -name '*.js' | xargs sed -i "s|$1|$2|g"
+  find $DEPLOYMENT_PATH/ -name '*.html' | xargs sed -i "s|$1|$2|g"
+}
+replace REACT_APP_API_VERSION "${REACT_APP_API_VERSION}"
```

Comment:
This line here will replace REACT_APP_API_VERSION with the actual version... say "19.2". Because in the webpack.config.common.js both the keyname && the env variable name are "REACT_APP_API_VERSION" isn't that going to result in

`19.2: 19.2`

instead of 

`REACT_APP_API_VERSION: 19.2
`
??

Reply:
Oh yes I see what you mean. In the content-editor (which is where I pinched this script from!) the variables are named differently, for example `apiStagingServer` using `REACT_APP_PREVIEW_HOST` env var.